os/include/net/lwip/lwipopts.h

```diff
@@ -63,6 +63,7 @@
  * In this platform, DHCP client and server are supported through netutils
  * Instead of using lwIP DHCP, please use netutils dhcpc and dhcpd
  */
+#define ENABLE_LOOPBACK					1
```

Comment:
Do we need this macro, as it depends on CONFIG_NET_LWIP_LOOPBACK_INTERFACE, which you have set to y in your defconfig above? Please check and confirm.